src/frontend/components/DashboardVideoLiveJitsi/utils.ts

```diff
@@ -92,6 +92,11 @@ export const initializeJitsi = async (
     ...live.live_info.jitsi.config_overwrite,
   };
 
+  if (isInstructor) {
+    configOverwrite.toolbarButtons?.push('participants-pane');
```

Comment:
Why not changing the `toolbarButtons` variable directly ?

Reply:
Because I don't know what will come in `live.live_info.jitsi.config_overwrite,` which may override `toolbarButton`.

Reply:
BTW, regarding @RVassili comment, I changed that too.